src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/AggregationIT.java

```diff
@@ -582,6 +582,69 @@ public void topHitTest_WithExclude() throws IOException {
 
     //region not migrated
 
+    // script on metric aggregation tests. uncomment if your elastic has scripts enable (disabled by default)
```

Comment:
This batch was not migrated because we didn't have painless module enabled then, I just moved them here so that we don't lose any tests. Me or someone else can update/uncomment later in a separate PR.